js/newsList.js

```diff
@@ -0,0 +1,216 @@
+import { category, news_by_category } from "../assets/news.js";
+import { listSubMouseClick } from "./subscribe.js";
+import presses from "../assets/light-media.js";
+
+let now_category = category[0];
+const total_pages = {};
+let page_count = {};
+category.forEach(item => {
+  total_pages[item] = news_by_category[item].length;
+  page_count[item] = 0;
+});
+
+function getNews(category) {
+  return news_by_category[category];
+}
+
+function drawArrow() {
+  if (page_count[now_category] === total_pages[now_category] - 1) {
+    hideArrow("right");
+    showArrow("left");
+  } else if (page_count[now_category] === 0) {
+    hideArrow("left");
+    showArrow("right");
+  } else {
+    showArrow("right");
+    showArrow("left");
+  }
+}
```

Comment:
맨 처음부터 둘다 화살표를 보여준 다음에 0이거나 마지막 페이지일 때만 각각 왼쪽, 오른쪽 화살표를 숨겨 보면 어떨까요?

```suggestion
function drawArrow() {
  showArrow('left');
  showArrow('right');
  if (page_count[now_category] === 0) {
    showArrow('left');
  } else if (page_count[now_category] === total_pages[now_category] - 1) {
    showArrow('right');
  }
}

```